packages/desktop-client/src/components/mobile/transactions/TransactionEdit.jsx

```
@@ -461,82 +467,94 @@
+      return payee?.transfer_acct && accountsById?.[payee.transfer_acct];
+    },
+    [accountsById, getPayee],
+  );
 
-  const isBudgetTransfer = trans => {
-    const transferAcct = trans && getTransferAcct(trans);
-    return transferAcct && !transferAcct.offbudget;
-  };
+  const getPrettyPayee = useCallback(
+    trans => {
+      if (trans && trans.is_parent) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)